operate/client/src/modules/types/operate.ts

```diff
@@ -55,6 +55,7 @@ interface ProcessInstanceEntity {
   processId: string;
   processName: string;
   processVersion: number;
+  processVersionTag?: string | null;
```

Comment:
Looking at the network tab, the v1 API already sent this property. So I added it here to be able to cleanup the store dependency in _ListView/InstancesTable/index.tsx_ (v1 table) as well.